### PR TITLE
tooltips

### LINKED ISSUE #36: [FEATURE] Add tooltips for experiment settings in the UI
### 🚀 Feature

Let's add tooltips for experiment settings in the UI

### Motivation

Quicker to look up than the `.md` file

### PR DESCRIPTION
**Pull request by pascal-pfeiffer**
adds tooltips to all experiment settings

closes #36

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,12 +105,13 @@ With H2O LLM Studio, training your large language model is easy and intuitive.
...
 ### Starting an experiment
-H2O LLM Studio provides various parameters to set for a given experiment, with some of the most important being:
+H2O LLM Studio allows to tune a variety of parameters and enables fast iterations to be able to explore different hyperparameters easily.
+The default settings are chosen with care and should give a good baseline. The most important parameters are:
 
 - **LLM Backbone**: This parameter determines the LLM architecture to use.
 - **Mask Prompt Labels**: This option controls whether to mask the prompt labels during training and only train on the loss of the answer.
 - **Hyperparameters** such as learning rate, batch size, and number of epochs determine the training process.
-An overview of all parameters is given in the [parameter description](docs/parameters.md).
+Please consult the tooltips of each hyperparameter to learn more about them.
```
Comment (psinger) [inline README.md:114]: Let's also link to the folder here, not everyone checks the GUI.

**Review by psinger — CHANGES_REQUESTED** [review]
Thanks!

The following tooltips are missing:

- Validation Dataframe
- Sync Batch Normalization
- Use Fsdp

Shall we remove the sync BN? Not too useful for Transformers I guess.

And shall we also have some sort of developer guide at some point that one needs to add this tooltip etc. But not too important at this point.

**Comment by pascal-pfeiffer** [comment]
> Thanks!
> 
> The following tooltips are missing:
> 
> * Validation Dataframe
> * Sync Batch Normalization
> * Use Fsdp
> 
> Shall we remove the sync BN? Not too useful for Transformers I guess.
> 
> And shall we also have some sort of developer guide at some point that one needs to add this tooltip etc. But not too important at this point.


Thank you for the review, I removed BN sync and added the two missing tooltips. 
If a tooltip doesn't exist, it will just not be shown in the UI, so I don't think that it is super important to have a developer guide for this matter.

**Review by psinger — APPROVED** [review]
ty!

**Pull request merged**